style.css

```diff
@@ -0,0 +1,66 @@
+/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+
+header {
+
+}
```

Comment:
Can't see that you have a header element in your HTML so you can just remove this